app/src/main/res/layout/simple_list_item_1.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@android:id/text1"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:gravity="center_vertical|right"
+    android:textSize="18sp"/>
```

Comment:
This is going to be tough. Text appearance used by system's simple_list_item_1 shows same text size as the phone text field on my device. After doing some tests maybe I found a better way. Replace the whole file with this:

```xml
<?xml version="1.0" encoding="utf-8"?>
<TextView
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:id="@android:id/text1"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:textAppearance="?android:attr/textAppearanceLarge"
    android:textColor="?android:attr/textColorSecondary"
    android:gravity="center_vertical|end" />
```

And also add this to phone_number EditText in number_validation.xml:

```
android:textAppearance="?android:attr/textAppearanceLarge"
```

This should make things equal (and bigger, I needed to do it anyway). I did a test with an older Android version (5), I'll do some other tonight, but I think this might work.